drivers/sensor/melexis/MLX90394/mlx90394.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2024 Florian Weber <Florian.Weber@live.de>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_MLX90394_MLX90394_H_
+#define ZEPHYR_DRIVERS_SENSOR_MLX90394_MLX90394_H_
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/rtio/rtio.h>
+
+#include "mlx90394_reg.h"
+
+enum {
+	WOC = SENSOR_ATTR_PRIV_START
+};
+/* Time it takes to start-up the device and switch to powerdown mode (after powercycle or soft
+ * reset)*/
+#define MLX90394_STARTUP_TIME_US 400
+
+/* conversion time of one single axis for different filter orders if osr is enabled the value has
+ * to be doubled */
+static const int32_t MLX90394_CONVERSION_TIME_US_AXIS[] = {111, 170,  270,  490,
+							   910, 1770, 3470, 6890};
+
+/* conversion time per axis dependent of filter order*/
+
+/* DSP Time per measurement first index:temperature sensor count ( 0..1 ) second index:
+ * magnetic axis count (0..3) in us */
+static const int32_t MLX90394_DSP_TIME_US[2][4] = {{0, 27, 50, 73}, {20, 63, 86, 110}};
+
+/* Conversion values */
+#define MLX90394_HIGH_RANGE_MICRO_GAUSS_PER_BIT       INT64_C(15000)
+#define MLX90394_HIGH_SENSITIVITY_MICRO_GAUSS_PER_BIT INT64_C(1500)
+#define MLX90394_MICRO_CELSIUS_PER_BIT                INT64_C(20000)
+
+/* values for setting SENSOR_ATTR_FULL_SCALE */
+#define MLX90394_ATTR_FS_HIGH_G INT32_C(500)
+#define MLX90394_ATTR_FS_LOW_G  INT32_C(50)
+
+struct mlx90394_data {
+	struct __attribute__((__packed__)) {
```

Comment:
Use `__packed`

Reply:
Thanks for the very early review. 
accepted and done